halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/admin/PublishBomCommand.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.admin;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.command.v1.NestableCommand;
+import com.netflix.spinnaker.halyard.cli.services.v1.Daemon;
+import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiUi;
+import lombok.AccessLevel;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Parameters()
+public class PublishBomCommand extends NestableCommand {
+  @Getter(AccessLevel.PUBLIC)
+  private String commandName = "bom";
+
+  @Getter(AccessLevel.PUBLIC)
+  private String description = "Publish a Bill of Materials (BOM).";
+
+  @Parameter(
+      names = "--bom-path",
+      required = true,
+      description = "The path to the BOM owning the artifact to publish."
+  )
+  private String bomPath;
+
+
+  @Override
+  protected void executeThis() {
+    Daemon.publishBom(bomPath);
+    AnsiUi.success("Published your BOM.");
```

Comment:
Feels redundant, since it's kinda long (when expanded) and it's written in plaintext in the command